packages/cli/src/local.js

```diff
@@ -0,0 +1,30 @@
+import { resolve, join, parse } from 'path'
+import { readdirSync, existsSync } from 'fs'
+import { requireModule } from './utils'
+import NuxtCommand from './command'
+
+const filterCommands = (dir) => {
+  return readdirSync(dir).filter(c => c.endsWith('.js'))
+}
+
+export function getLocalCommands() {
```

Comment:
These utilities seem are going to be a standard way for a _group_ of commands. What do you think of introducing `CommandGroup` interface class with public methods including `exists(cmd)` / ``get(cmd)`? (Useful for modules that expose more than one command). PS: We need to somehow add a namespace support so that 3rd party modules are restricted to only use their own. (Except builtIn ones)

Reply:
Hmm, take a look at:

https://github.com/nuxt/nuxt.js/tree/feat/module-commands/packages/cli/src/command
https://github.com/nuxt/nuxt.js/blob/feat/module-commands/packages/cli/src/run.js

WDYT? That's from my other branch (evolved from this one).

If you're OK with this PR, I can address these changes in that next one :)